### ISSUE TITLE
[Errno 2] No such file or directory: '/app/node_modules/.bin/lessc'

### ISSUE THREAD

**Issue by willkg** [mozilla/standup#240]
I get this error when trying to run Standup locally in a docker container via `make run`:

```
[Errno 2] No such file or directory: '/app/node_modules/.bin/lessc'
```

Running `make build-base` works fine and I see the modules getting installed and they get installed into `/app`, but then that gets stomped on and we lose the node stuff.

This issue covers fixing the docker file so that the node dependencies don't also have to be installed on the host.

**Comment by willkg** [comment]
First off, I have no idea if the explanation makes any sense. If it doesn't, that's my fault.

@pmac suggested switching it to `npm install -g` and then tweaking paths. I'll try that.

**Comment by pmac** [comment]
I think this is now fixed via #243
